src/main/java/com/iexec/common/chain/validation/EthereumNonZeroAddressValidator.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.common.chain.validation;
+
+import com.iexec.common.utils.BytesUtils;
+import com.iexec.common.utils.EthAddress;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EthereumNonZeroAddressValidator
```

Comment:
Could this be tested with unit tests in order to show valid and invalid use cases ?